anaconda.py

```diff
@@ -718,7 +718,7 @@ def _earlyExceptionHandler(ty, value, traceback):
     from pyanaconda.modules.common.constants.services import TIMEZONE
     timezone_proxy = TIMEZONE.get_proxy()
 
-    if can_touch_runtime_system("initialize time", touch_live=True):
+    if conf.system.can_adjust_time_live:
```

Comment:
We are going to use can_configure_hwclock_time.

Reply:
The rules for time are replaced with `can_set_hardware_clock`, `can_set_system_clock` and `can_initialize_system_clock`. The last one is a temporary solution, because I don't think that it makes sense.